From: Thomas Koch <thomas.koch@ymc.ch>
Subject: [PATCH] debian/solr-example-solrconfig

Two problems with the configuration from examples/

- Contains <lib> tags that fail
- The solr index directory is not FHS compliant

Signed-off-by: Thomas Koch <thomas.koch@ymc.ch>

---
 example/solr/conf/solrconfig.xml |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

Index: solr/example/solr/conf/solrconfig.xml
===================================================================
--- solr.orig/example/solr/conf/solrconfig.xml	2011-06-20 17:32:12.000000000 +0200
+++ solr/example/solr/conf/solrconfig.xml	2011-07-05 10:43:35.669649074 +0200
@@ -67,20 +67,20 @@
        the classpath, this is useful for including all jars in a
        directory.
     -->
-  <lib dir="../../contrib/extraction/lib" />
+  <!--lib dir="../../contrib/extraction/lib" /-->
   <!-- When a regex is specified in addition to a directory, only the
        files in that directory which completely match the regex
        (anchored on both ends) will be included.
     -->
-  <lib dir="../../dist/" regex="apache-solr-cell-\d.*\.jar" />
+  <!--lib dir="../../dist/" regex="apache-solr-cell-\d.*\.jar" />
   <lib dir="../../dist/" regex="apache-solr-clustering-\d.*\.jar" />
-  <lib dir="../../dist/" regex="apache-solr-dataimporthandler-\d.*\.jar" />
+  <lib dir="../../dist/" regex="apache-solr-dataimporthandler-\d.*\.jar" /-->
 
   <!-- If a dir option (with or without a regex) is used and nothing
        is found that matches, it will be ignored
     -->
-  <lib dir="../../contrib/clustering/lib/" />
-  <lib dir="/total/crap/dir/ignored" /> 
+  <!--lib dir="../../contrib/clustering/lib/" />
+  <lib dir="/total/crap/dir/ignored" /-->
   <!-- an exact path can be used to specify a specific file.  This
        will cause a serious error to be logged if it can't be loaded.
     -->
@@ -95,7 +95,7 @@
        replication is in use, this should match the replication
        configuration.
     -->
-  <dataDir>${solr.data.dir:}</dataDir>
+  <dataDir>/var/lib/solr/data</dataDir>
 
 
   <!-- The DirectoryFactory to use for indexes.
